src/NumberTheory/QuadFormAndIsom/types.jl

```diff
@@ -209,3 +209,58 @@ endowed with an isometry.
     return new(Vf, Lb, f, n)
   end
 end
+
+@doc raw"""
+    TorQuadModuleWithIsom
+
+A container type for pairs $(T, f)$ consisting of a torsion quadratic module
```

Comment:
```suggestion
A type for pairs $(T, f)$ consisting of a torsion quadratic module
```
Every type is more or less a container.